src/emysql.erl

```diff
@@ -324,7 +322,7 @@ add_pool(PoolId, Size, User, Password, Host, Port, Database, Encoding) ->
       Database :: string(),
       Encoding :: utf8 | latin1 | {utf8, utf8_unicode_ci} | {utf8, utf8_general_ci},
```

Comment:
Line 323 is too long:       Encoding :: utf8 | latin1 | {utf8, utf8_unicode_ci} | {utf8, utf8_general_ci},.